src/services/npdi.ts

```diff
@@ -1,23 +1,22 @@
 import path from 'node:path';
-import fs from 'fs-extra';
 import express from 'express';
 import subdomain from 'express-subdomain';
+import { fileErrCallback } from '@/util';
 
 const npdi = express.Router();
 
 npdi.get('/p01/data/1/:titleHash/:dataID/:fileHash', (request, response) => {
 	const { titleHash, fileHash } = request.params;
 	const contentPath = path.normalize(`${__dirname}/../../cdn/content/encrypted/${titleHash}/${fileHash}`);
 
-	if (fs.existsSync(contentPath)) {
-		response.set('Content-Type', 'applicatoin/octet-stream');
-		response.set('Content-Disposition', 'attachment');
-		response.set('Content-Transfer-Encoding', 'binary');
-		response.set('Content-Type', 'applicatoin/octet-stream');
-		response.sendFile(contentPath);
-	} else {
-		response.sendStatus(404);
-	}
+	response.sendFile(contentPath, {
+		headers: {
+			// The misspelling here is intentional, it's what the official server sets
+			'Content-Type': 'applicatoin/octet-stream',
+			'Content-Disposition': 'attachment',
+			'Content-Transfer-Encoding': 'binary',
```

Comment:
Minor nit. We don't use trailing commas at the end of lists/objects

```suggestion
			'Content-Transfer-Encoding': 'binary'
```

Reply:
Similar with the imports, I'll update config so this is automated 

What's the thoughts on whether to use Prettier? Personally, I like it because it enforces opinionated styling and doesn't take a lot of effort to get right

There's the alternative of using the [eslint stylistic plugin](https://eslint.style/packages/js) which is more configurable, but imo will lead to more discussions/arguments over formatting which is not critical

Reply:
I remember using Prettier in the past and, honestly, kinda just straight up hating the output. I also didn't like how it wasn't super customizable. I respect it's place in the ecosystem and I 100% understand that it's *supposed* to be opinionated, but I'm not sure it's a good fit here

Reply:
Sure, I'll look at the stylistic plugin then

They'll have pretty sensible defaults, so it'll just be a case of trial and error to get it dialed in - if you can point me to any style guides we have then that'll help!

Once we've got it where we want it, we'll be able to setup a [sharable config](https://eslint.org/docs/latest/extend/shareable-configs) which we can use on all the JS repos moving forward